images/win/windows2016.json

```diff
@@ -138,7 +139,8 @@
             "type": "powershell",
             "inline": [
                 "setx ImageVersion {{user `image_version` }} /m",
-                "setx ImageOS {{user `image_os` }} /m"
+                "setx ImageOS {{user `image_os` }} /m",
+                "setx AGENT_TOOLSDIRECTORY {{user `agent_tools_directory` }} /m"
```

Comment:
Maybe move all this inline-block to https://github.com/actions/virtual-environments/blob/main/images/win/scripts/Installers/Initialize-VM.ps1?

Reply:
Hi, @ryanspletzer 
Could you please merge the latest changes from main branch and  put  `setx AGENT_TOOLSDIRECTORY` near the line https://github.com/actions/virtual-environments/blob/1eb504f4d48c428bf25a4c0fce604e98d8ac6f4c/images/win/scripts/Installers/Initialize-VM.ps1#L36 ?

Reply:
Yep, done and done.